### DISCUSSION TITLE
Locale Setting for date format independently from the browser

### DISCUSSION THREAD

**Discussion by natrius** [firefly-iii/firefly-iii#6858] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

- I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has been discussed before, but the issue is locked so i cannot comment there. See https://github.com/firefly-iii/firefly-iii/issues/2824

I would like to choose the date-format of my own liking. I like to set my browser and programs to english but i don't like the MDY format - the problem is, i have to use that all the time when entering a new entry. 
So, worldwide https://en.wikipedia.org/wiki/ISO_8601 is respected and according to https://en.wikipedia.org/wiki/Date_format_by_country DMY is the most common/widespread variant. 

Screenshot:
![image](https://user-images.githubusercontent.com/7243447/212286507-5b28dc96-0317-4454-a7c0-e9d264bd7a71.png)

Another thing - this would also prevent a potential mistake when entering when using a different browser. 

### Solution

Extra point in "Locale settings" and add something like "Date format". Let the user Choose between DMY, YMD and MDY.

**Comment by JC5** [comment]
The date format is decided upon by the browser, not by me. I simply use `type="date"`.

**Reply by natrius** [reply]
Thanks for the answer. I think it should be possible to be set individually. For example in the preferences a extra setting for date format
![image](https://user-images.githubusercontent.com/7243447/215438646-61871bab-7045-4ef8-a3a7-27a5a658567f.png)
would be good. Even if i prefer to set the browser language to english i would like a 'normal' (for me :D ) date-formatting.

**Reply by JC5** [reply]
Yes I think so to. Talk to your browser maker.
